### PR TITLE
Replace Karapace's website with a landing page and some docs

### PR DESCRIPTION
**Pull request by lornajane**
# About this change - What it does
Adds a simple sphinx setup in a `website/` folder that builds a static HTML website. Also adds styles, templates, and content.

Fixes: #381 

# Why this way
We already have Sphinx sites inside the business, and Python dependencies in this project.

### CODE REVIEW

**Comment by lornajane** [comment]
With help from my friend (thanks Ioan!)
[Karapace.zip](https://github.com/aiven/karapace/files/9192341/Karapace.zip)
, I got some static assets to use as a template

**Comment by lornajane** [comment]
Looking okay on desktop (I think!) but the navigation menu does not work for small screens, so it will need attention

File: website/source/_templates/page.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+						<small class="trademark">Apache Kafka is either a registered trademark or a trademark of the Apache Software Foundation in the United States and/or other countries.</small>
+					</div>
+				</div>
+                {% endblock footer %}
+                </div>
+            </footer>
+        </div>
+    </div>
+</div>
+{%- endblock %}
```
Comment (blomqma) [inline website/source/_templates/page.html:96]: nit: newline

**Comment by lornajane** [comment]
replaced by #489

**Pull request closed without merging**